tests/semantic/witness_check/witness_check.dl

```diff
@@ -0,0 +1,9 @@
+.decl A(x:number, y:symbol)
```

Comment:
Does this test add anything not tested for in witness_check2?
If no, then remove it. If yes, add the header, as for witness_check2.